manifests/alertmanager/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/path: '/alertmanager/metrics'
+  labels:
+    name: alertmanager
+  name: alertmanager
+  namespace: monitoring
+spec:
+  selector:
+    app: alertmanager
+  #type: NodePort
```

Comment:
`NodePort` is needed for Minikube as long as no ingress is setup. I could access the Web-UI via the NodePort after disabling `-web.external-url`.